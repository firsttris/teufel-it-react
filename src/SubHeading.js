import React, { Component } from 'react';

class SubHeading extends Component {
  render(props) {
    return (
      <div className="col">
        <div className="text-uppercase" style={{ fontSize: '30px', letterSpacing: '10px' }}>{this.props.title}</div>
        <p>{this.props.text}</p>
      </div>
    );
  }
}

export default SubHeading;
