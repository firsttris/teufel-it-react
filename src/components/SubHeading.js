import React, {Component} from 'react';
import {FormattedMessage} from 'react-intl';

export default class SubHeading extends Component {
    render() {
        return (
            <div className="row py-3" style={this.props.style}>
                <div className="col">
                    <div className="text-uppercase"
                         style={{fontSize: '30px', letterSpacing: '10px'}}>
                        <FormattedMessage id={this.props.title}/>
                    </div>
                    <FormattedMessage id={this.props.text}/>
                </div>
            </div>
        );
    }
}
