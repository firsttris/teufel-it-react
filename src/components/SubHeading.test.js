
import React from 'react';
import SubHeading from './SubHeading';
import renderer from 'react-test-renderer';

test('Display Subheading', () => {
  const component = renderer.create(
    <SubHeading title="Test Title" text="Subheading Test Title"></SubHeading>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});