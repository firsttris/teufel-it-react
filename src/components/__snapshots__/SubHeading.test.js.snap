// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display Subheading 1`] = `
<div
  className="col"
>
  <div
    className="text-uppercase"
    style={
      Object {
        "fontSize": "30px",
        "letterSpacing": "10px",
      }
    }
  >
    Test Title
  </div>
  <p>
    Subheading Test Title
  </p>
</div>
`;
